tinyagentos/routes/user_memory.py

```
@@ -213,16 +232,20 @@ async def migrate_to_taosmd(request: Request):
                     try:
-                        await client.post(
+                        one = await client.post(
                             f"{base}/ingest",
                             json={"text": item["text"], "agent": _TAOSMD_AGENT,
                                   "metadata": item["metadata"]},
                         )
-                        ingested += 1
+                        if one.status_code < 400:
+                            ingested += 1
                     except Exception:
                         pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)